andrew_eye_stream_data.py

```diff
@@ -0,0 +1,50 @@
+from talon import app, ui
+from talon_plugins.eye_mouse import tracker
+from talon.track.geom import EyeFrame
+
+main = ui.main_screen()
+
+# Lightweight simple recording for eye tracking.
+class EyeDataStream:
+    def __init__(self):        
+      self.x_coords = []
+      self.y_coords = []
+
+    def on_gaze(self, b):
```

Comment:
What is the data type of `b`?

Reply:
It's some sort of custom data type that is being emitted by the tobii 4C directly.